full-service/src/bin/main.rs

```diff
@@ -274,11 +295,32 @@ fn validator_backed_full_service(
 
     let fog_ingest_verifier = config.get_fog_ingest_verifier();
     let logger2 = logger.clone();
+
+    let chain_id = config.peers_config.chain_id.clone();
+    let tx_sources: Option<Vec<String>> = config.peers_config.tx_source_urls.clone();
+    let peers: Option<Vec<String>> = match config.peers_config.peers.clone() {
+        None => None,
+        Some(peers) => Some(
+            peers
+                .iter()
+                .map(|peer_uri| peer_uri.url().clone().into())
+                .collect(),
+        ),
+    };
+
+    let network_setup_config = NetworkSetupConfig {
+        offline: config.offline.clone(),
+        chain_id,
+        peers,
+        tx_sources,
+    };
+
```

Comment:
Could we move this to the higher level function (main) so it isn't duplicated and just pass it in as part of the params of `consensus_backed_` and `validator_backed_full_service`?